modules/api/lambda/src/handlers/_/index.ts

```diff
@@ -45,7 +45,7 @@ export interface HandlerCallbackEvent<
   path: string                         /* Event path */
   pathParameters: TParameters          /* Event path parameters */
   headers: {                           /* Event headers */
-    [name: string]: string
+    [name: string]: string | undefined
```

Comment:
Headers can be undefined, so accepting this change we'll lose type safety.

Reply:
This changes is because of this:
![image](https://user-images.githubusercontent.com/1132274/126068260-e38eb5b0-7ff2-4f37-a42f-d59ee145fb58.png)

The package `@types/aws-lambda` now have this signature at `@types/aws-lambda/trigger/api-gateway-proxy.d.ts/APIGatewayProxyEventHeaders`.